easybuild/easyconfigs/g/GPAW/GPAW-1.5.2-slurm-submit-fix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/gpaw/cli/sbatch.py b/gpaw/cli/sbatch.py
```

Comment:
Please add description and author info to the patch, and don't forget to update the checksum for it in the easyconfigs.

Reply:
Done